app/web/routes.py

```diff
@@ -248,7 +254,8 @@ def history():
     timeout=300,
 )
 def sla():
-    time_now = datetime.now()
+    time_now_str = get_utc_timestamp()
+    time_now = datetime.strptime(time_now_str, '%Y-%m-%d %H:%M:%S.%f%z')
```

Comment:
why doing conversion and back conversion if it should be enough to use  datetime.now(timezone.utc) ?

Reply:
Cleaned up